third_party/move/tools/move-cli/src/base/test.rs

```diff
@@ -13,14 +13,16 @@ use move_compiler::{
     PASS_CFGIR,
 };
 use move_coverage::coverage_map::{output_map_to_file, CoverageMap};
-// if windows
-use move_package::compilation::compiled_package::unimplemented_v2_driver;
-use move_package::{compilation::build_plan::BuildPlan, BuildConfig, CompilerConfig};
+use move_package::{
+    compilation::{build_plan::BuildPlan, compiled_package::unimplemented_v2_driver},
+    BuildConfig, CompilerConfig,
+};
 use move_unit_test::UnitTestingConfig;
 use move_vm_test_utils::gas_schedule::CostTable;
 // if unix
 #[cfg(target_family = "unix")]
 use std::os::unix::prelude::ExitStatusExt;
+// if windows
```

Comment:
These would actually be clearer with blank lines instead of the // if unix and // if windows comments.  The #[cfg ...] are pretty self-documenting.

Reply:
That is not me. I'm just correcting a formatting mistake and bring this comment back to where it was before the last PR. My last PR accidentally messed this up, and Greg noticed and asked to fix.

Reply:
Ok.  Though these are pretty useless comments.